demos/common/transport/transport_tls_socket_using_mbedtls.c

```diff
@@ -830,7 +830,7 @@ int32_t TLS_Socket_Recv( NetworkContext_t * pxNetworkContext,
         ( lTLSStatus == MBEDTLS_ERR_SSL_WANT_WRITE ) )
     {
         LogDebug( ( "Failed to read data. However, a read can be retried on this error. "
-                    "mbedTLSError= %s : %s.",
+                    "mbedTLSError[%d]= %s : %s.", lTLSStatus,
```

Comment:
nit: why have this be basetype_t in the disconnect but int32_t here?

Reply:
Issue in source where it is taken from. For consistency I will make it int32_t.